meson.build

```diff
@@ -47,6 +47,7 @@ cflags = [
     '-Wwrite-strings',
     '-Wno-documentation-deprecated-sync',
     '-Wno-pedantic',
+    '/utf-8',
```

Comment:
Feel free to cherry-pick this to master if you'd like.

Reply:
There is nothing that requires it atm, let’s wait to merge the MR.